components/camel-debezium/camel-debezium-common/camel-debezium-common-component/pom.xml

```diff
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
-            <version>3.0.2</version>
+            <version>3.3.2</version>
```

Comment:
Is it based on Kafka 3.3.2?

Reply:
thanks, checking it more carefully, 2.1.2.Final is actually based on [kafka 3.3.1](https://github.com/debezium/debezium/blob/v2.1.2.Final/pom.xml#L96) (we recently upgraded to 3.4.0, but it will be in 2.2.0 which is not released yet)